docs/basics/101-102-populate.rst

```diff
@@ -87,7 +87,9 @@ does not store the downloaded books in its history yet. Let's change this by
 (:manpage:`datalad-save` manual).
 
 This time, it is your turn to specify a helpful :term:`commit message`
-with the ``-m`` option:
+with the ``-m`` option (although the DataLad command is :command:'datalad save', we talk
+about commit messages because :command:'datalad save' ultimatively uses the command
+:command:'git commit' to do its work):
```

Comment:
In order to use any markup like ``:command:``, those ticks need to be backticks

```suggestion
with the ``-m`` option (although the DataLad command is :command:`datalad save`, we talk
about commit messages because :command:`datalad save` ultimatively uses the command
:command:`git commit` to do its work):
```

Reply:
Thanks. Did not see that properly. Should update my font (or my glasses).